google/ggrc-core
# GGRC-4838 Add pagination for TGT list

# Dependencies

This PR is `on hold` until the dependencies are merged:

- [x] GGRC-4843


# Issue description

Add pagination for the Task list instead of the long list with Tasks.

# Steps to test the changes

1. Open any workflow.
2. Open Setup tab.
3. Open any Task Group.
Expected result: All tasks should be showed though the pagination.

# Solution description

Was reused the way for the pagination, which was applied for showing the proposals (go to Control info pane to see the tab).

# Sanity checklist




# PR Review checklist
